forum/qa-plugin/tips-widget/tips-widget-admin.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+class tips_widget_admin
+{
+	function option_default($option)
+	{
+		switch ($option)
+		{
+			case 'tips-enable':
+				return 0;
+			case 'tips-widget-content':
+				return <<< TAG
```

Comment:
Cała ta treść powinna znaleźć się w osobnym pliku, a nie śmiecić tu niepotrzebnie

Reply:
Mogę to dać, nie wiem... do JSON-a, ale ten kod jest tak mikroskopijny, że takie działanie jedynie rozwlekałoby czytanie takiego kodu, gdyż trzeba by było przechodzić po plikach w szukaniu domyślnych wartości.

Reply:
To daj. Olewanie szczegółów, nawet jeśli są mikroskopijne, to przyzwolenie na olewanie swojej pracy i czasu ludzi, którzy muszą to przeglądać i tłumaczyć podstawy programowania